pages/en/lb4/Sequence.md

```diff
@@ -86,18 +86,16 @@ class MySequence extends DefaultSequence {
 }
 ```
 
-In order for LoopBack to use your custom sequence, you must register it on any
-applicable `Server` instances before starting your `Application`:
+In order for LoopBack to use your custom sequence, you must register it
+before starting your `Application`:
 
 ```js
 import {RestApplication, RestServer} from '@loopback/rest';
 
 const app = new RestApplication();
 
-// or
 (async function start() {
-  const server = await app.getServer(RestServer);
-  server.sequence(MySequence);
+  app.sequence(MySequence);
```

Comment:
Ditto, please move this line out of async `start()` function and consider removing the start function completely.